android/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "29.0.2"
-        minSdkVersion = 19
+        minSdkVersion = 29
```

Comment:
Doesn't this mean our app would only more be allowed to be used with Android device using SDK 29 (Android 10)?

Reply:
The main reason was that **react-native-background-upload** need a minimum of  `minSdkVersion = 21` for it to work

Reply:
Why not take SDK 21 as `minSdkVersion` then? ;-)